modules/local/gfastats.nf

```diff
@@ -0,0 +1,25 @@
+process GFASTATS {
+    tag "${meta.id}"
+    label 'process_single'
+
+    conda (params.enable_conda ? "bioconda::gfastats=1.3.5" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gfastats:1.3.5--hd03093a_0' :
+        'quay.io/biocontainers/gfastats:1.3.5--hd03093a_0' }"
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    path("*stats"), emit: stats
+    path "versions.yml",     emit: versions
+
+    script:
+    def args = task.ext.args ?: ''
+    """
+    gfastats ${fasta} ${args} > ${fasta}.stats
```

Comment:
What is gfastats? There is an nf-core module for it: https://github.com/nf-core/modules/blob/master/modules/nf-core/gfastats/main.nf. Can we use that?